explore/src/main/scala/explore/tabs/ObsTabContents.scala

```diff
@@ -532,7 +532,7 @@ object ObsTabContents {
               .runAsync
       }
       // Shared obs conf (posAngle/obsTime)
-      .useStateViewWithReuse(ObsConfiguration(PosAngle.Default, LocalDateTime.now()))
+      .useStateViewWithReuse(ObsConfiguration(PosAngle.Default, new js.Date().toLocalDateTime))
```

Comment:
This is a good observation. Now this time is always going to be in UTC, maybe it makes more sense to make that explicit?
In either case the use of `now` is not ideal, but it is done in the interim as a solution to not having this value coming from the odb

Reply:
> This is a good observation.

Actually, the app was crashing for me since I'm in a different time zone :D

> Now this time is always going to be in UTC

`new js.Date()` produces a `Date` with the local time actually.

Reply:
After our Slack discussion, I'm changing the value to store an `Instant`.